src/NServiceBus.Core/Pipeline/PipelineCollection.cs

```diff
@@ -8,13 +8,22 @@ namespace NServiceBus
 
     class PipelineCollection
     {
-        public PipelineCollection(IEnumerable<TransportReceiver> pipelines)
+        public PipelineCollection()
```

Comment:
wouldn't it be cleaner to have something like a dedicated "dummy" collection for the send-only use case to avoid having 2 ctors and null checks in start/stop? (or `Add` pipelines to the collection instead of passing them in the ctor?)